src/models/genre/id.yaml

```
@@ -1,8 +1,12 @@
 type: object
 required:
   - id
-description: "The identifier for the genre"
 properties:
   id:
+    description: |
+      The unique identifier for the genre. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)